pkg/selinux/setroubleshoot-client.js

```diff
@@ -35,9 +35,10 @@ var busNameFixit = "org.fedoraproject.SetroubleshootFixit";
 var dbusInterfaceFixit = busNameFixit;
 var dbusPathFixit = "/org/fedoraproject/SetroubleshootFixit/object";
 
-client.init = function() {
+client.init = function(capabilitiesChangedCallback) {
```

Comment:
> You're right, this is the most pragmatic solution. Especially when it's only temporary.

I added a "HACK" note and linked to the bugzilla entry, we can track the change when the API has arrived in RHEL. https://bugzilla.redhat.com/show_bug.cgi?id=1306700